packages/big-design/src/components/Table/spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
Looks like we should actually update the types on `columns` and `items` in this file
```ts
interface SimpleTableOptions {
  // ...
  columns?: Array<TableColumn<Record<string, unknown>>>;
  items?: Array<Record<string, unknown>>;
  // ...
}
```